### DISCUSSION TITLE
sBTC Testnet: Update on the v0.1 Developer Release Milestone

### DISCUSSION THREAD

**Discussion by andrerserrano** [stacks-network/stacks-core#3904] [Blockchain]
Hi everyone,

I wanted to share an update on the decision to release sBTC on a public testnet. The testnet has been in development since Q1 2023. We have spent the past few months refining the user experience and preparing a public developer release. This post will go into more detail on the goals of this release on the feature set that users should expect.

**TLDR:**
- We plan to release a public sBTC testnet within 4-6 weeks (mid Oct)
- The goal for this release to allow early access to developers and gather critical feedback for the sBTC working group
- This approach optimizes for getting Nakamoto launched on mainnet, which is a critical prereq for the sBTC launch, as soon as possible (safely)… while still balancing getting sBTC into developer hands ASAP

**What is the sBTC Developer Release?**
The [sBTC Developer Release](https://stacks-network.github.io/sbtc-docs/sbtc-releases/sbtc-dev.html) is a minimum-viable testnet deployment of sBTC. It will allow us to continue testing the deposit and withdrawal functionality, without the complexity of adding a decentralized set of signers to validate sBTC deposits/withdrawals. Instead, this release will use a central coordinator, which has two primary functions: (1) mint and burn sBTC (2) Fulfill peg-out requests. This version will be released on public Testnet only.

**Who is this release for?**
This early release is primarily for developers, to begin building momentum with builders and accelerating apps that will integrate sBTC. Go-to-market initiatives will focus on hackathons, workshops, user research testing and developer education.

**What is included in this release?**
The release will include an implementation of the sBTC bridge web and API product that supports the following essential features;
- Deposit bitcoin into sBTC tokens on the Stacks network
- Withdraw sBTC tokens back to bitcoin
- Transaction history tracking

**Deployment Plan:**
As part of our deployment plan, upcoming milestones have been scoped and tasks divided on the [Stack Core Eng](https://github.com/orgs/stacks-network/projects/63/views/10) project board.

The sBTC testnet optimizes for getting the sBTC contracts in developers hands asap, so we can bolster Nakamoto readiness which is crucial for sBTC. This release streamlines our deployment plan, allowing us to focus on delivering upcoming milestones faster and with greater alignment with business and product requirements. 

The deployment plan for the release is tracked in this [issue](https://github.com/stacks-network/sbtc-docs/pull/79#pullrequestreview-1600729097). 

**Note on terms**
This release is also known as the Developer Release, Version 0.1, and colloquially referred to as Alpha Romeo. Please note that the release known as ‘sBTC Mini’ has been deprecated.

**Comment by mijoco-btc** [comment]
Hey @andrerserrano do we have the contract id for Romeo ?

**Reply by friedger** [reply]
It is configurable in the Romeo engine binary.
